### ISSUE TITLE
dataset split

### ISSUE THREAD

**Issue by cancerwower** [zhouh115/DWT-FFC#2]
Can you make public the training set, validation set and test set corresponding to the comparative experiment of your article?  i would be very grateful！

**Comment by yaningjm** [comment]
Do you know how to train the model？
